bonsai/README.md

```diff
@@ -0,0 +1,65 @@
+## Overview
+
+Track request level metrics for your Bonsai clusters to:
+
+* Visualize the performance of your clusters
+* Correlate search performance with application performance
+* Create alerts
+
+![snapshot](https://raw.githubusercontent.com/omc/IntegrationTemplate/master/Bonsai/images/snapshot.png)
+
+## Setup
+
+Integrating your cluster with Datadog requires submitting your API key to the bonsai app.
+
+
+### Step 1 - Aquire API Key
+
+Navigate to https://app.datadoghq.com/account/settings#api and copy the API Key
```

Comment:
```suggestion
In Datadog, navigate to [Integrations --> API][1] and copy your API Key.
```

Reply:
Put links at the bottom of the page in a list:

`[1]: https://app.datadoghq.com/account/settings#api`